recipe/patches/0019-Add-CondaEcosystemModifyDllSearchPath.patch

```diff
@@ -1,456 +1,447 @@
-From 8b93cecf78873b72a0c1d862069fbf39f03c0774 Mon Sep 17 00:00:00 2001
-From: Ray Donnelly <mingw.android@gmail.com>
-Date: Fri, 25 Jan 2019 22:54:57 +0000
-Subject: [PATCH 19/19] Add CondaEcosystemModifyDllSearchPath()
-
-This changes the DLL search order so that C:\Windows\System32 does not
-get searched in before entries in PATH.
-
-Reviewed by Kai Tietz 7.2.2019
-
-Updated a bit to include other directories.
----
- Modules/main.c       | 387 ++++++++++++++++++++++++++++++++++++++++++-
- Python/dynload_win.c |   4 +
- 2 files changed, 390 insertions(+), 1 deletion(-)
-
-diff --git a/Modules/main.c b/Modules/main.c
-index b3fba94..38da7e2 100644
---- a/Modules/main.c
-+++ b/Modules/main.c
-@@ -16,6 +16,10 @@
- #  ifdef HAVE_FCNTL_H
- #    include <fcntl.h>
- #  endif
-+#  include <shlwapi.h>
-+#  include <string.h>
-+#  include <malloc.h>
-+#  include <libloaderapi.h>
- #endif
- 
- #ifdef MS_WINDOWS
-@@ -56,6 +60,8 @@ extern "C" {
-         } \
-     } while (0)
- 
-+char *CondaEcosystemGetWarnings();
-+
- #ifdef MS_WINDOWS
- #define WCSTOK wcstok_s
- #else
-@@ -1238,7 +1244,7 @@ pymain_header(_PyMain *pymain)
-         return;
-     }
- 
--    fprintf(stderr, "Python %s :: %s on %s\n", Py_GetVersion(), Anaconda_GetVersion(), Py_GetPlatform());
-+    fprintf(stderr, "Python %s :: %s on %s\n%s", Py_GetVersion(), Anaconda_GetVersion(), Py_GetPlatform(), CondaEcosystemGetWarnings());
-     if (!Py_NoSiteFlag) {
-         fprintf(stderr, "%s\n", COPYRIGHT);
-     }
-@@ -3017,9 +3023,388 @@ pymain_init(_PyMain *pymain)

```

Comment:
Need to skip this part because this builds on `0001-Add-Anaconda-Distribution-version-logic.patch`, which is applied neither on [3.7](https://github.com/conda-forge/python-feedstock/blob/3.7/recipe/meta.yaml#L17-L19) nor on [master](https://github.com/conda-forge/python-feedstock/blob/d0ab36792688c2531d6d89e5ad3f201af174094b/recipe/meta.yaml#L20-L22).